frontend/src/css/modules/base.scss

```diff
@@ -180,8 +180,16 @@ span.inline {
 
 /*main intro page*/
 
-.ev-md-container {
-    padding: 40px;
+.ev-md-dialog-content {
```

Comment:
@Obtin why you have deleted `ev-md-container`?

Reply:
this is because I changed the `ev-md-container` to 'ev-md-content' in template